raiden/transfer/views.py

```diff
@@ -397,15 +405,15 @@ def get_channelstate_settled(
     )
 
 
-def role_from_transfer_task(transfer_task: TransferTask) -> str:
+def role_from_transfer_task(transfer_task: TransferTask) -> TransferRole:
     """Return the role and type for the transfer. Throws an exception on error"""
     # pragma: no cover
     if isinstance(transfer_task, InitiatorTask):
-        return "initiator"
+        return TransferRole.INITIATOR
     if isinstance(transfer_task, MediatorTask):
-        return "mediator"
+        return TransferRole.MEDIATOR
     if isinstance(transfer_task, TargetTask):
-        return "target"
+        return TransferRole.TARGET
 
     raise ValueError("Argument to role_from_transfer_task is not a TransferTask")
```

Comment:
I would rather put a constant `role` attribute in the `TransferTask` subclasses than having this function.